experiments/models/sample.py

```diff
@@ -20,7 +20,8 @@ class Background(NameModel):
 
 
 class Sample(models.Model):
-    id = models.CharField(max_length=40, primary_key=True)
+    name = models.CharField(max_length=40, unique=True)
+    # TODO: make this a field on Measurement
     species = models.IntegerField(blank=True, null=True, choices=[
```

Comment:
Should it be defaulted to human (`Hca`)?

Reply:
I need to explicitly clarify that with Kenny